release/update-repository-version.sh

```diff
@@ -131,22 +130,45 @@ EOT
 	exit "$exit_code"
 }
 
-while getopts "hp" opt; do
-	case $opt in
-	h) usage 0 ;;
-	p) PUSH="true" ;;
-	esac
-done
-
-shift $((OPTIND - 1))
-
-repo=${1:-}
-new_version=${2:-}
-target_branch=${3:-}
-[ -n "${repo}" ] || (echo "ERROR: repository not provided" && usage 1)
-[ -n "${new_version}" ] || (echo "ERROR: no new version" && usage 1)
-[ -n "${target_branch}" ] || die "no target branch"
-
-pushd "$tmp_dir" >>/dev/null
-bump_repo "${repo}" "${new_version}" "${target_branch}"
-popd >>/dev/null
+repos=(
```

Comment:
Might be worth adding a comment explaining why `tests` isn't included (since it has stable releases).

Reply:
Added  a comment we branch test but there is not VERSION file there in packaging.